int8 direction # only move straight 1 = forwards; -1 = backwards
float32 speed         # m/s    (must be >Â 0)
---
bool success